app/src/main/res/values/strings.xml

```diff
@@ -412,6 +412,10 @@
     <string name="creation_date_reversed">Creation date (reversed)</string>
     <string name="alphabetic">Alphabetic</string>
     <string name="alphabetic_reversed">Alphabetic (reversed)</string>
+    <string name="welcome">Welcome to LibreTube</string>
+    <string name="choose_instance">Please choose an instance first!</string>
+    <string name="choose_instance_long">Please choose a Piped instance to use below. The Piped instance will act as a middle man between you and YouTube. These servers are located at different physical places (indicated with flags). Instances that are close to where you live, are likely faster than others. You can still change it later in the instance settings.</string>
+    <string name="registration_disabled">Registration disabled</string>
```

Comment:
Potential improvement.
```suggestion
    <string name="choose_instance_long">Please choose a Piped instance to use from below. The Piped instance will act as the middleman between you and YouTube. These instances are located at different physical locations - indicated by their country flag(s). Instances physically closer to you are likely faster than instances that are not. You can change the instance you use in the settings at any time.</string>
    <string name="registration_disabled">Registration disabled</string>
```
```suggestion
    <string name="choose_instance_long">Please choose a Piped instance to use from below. The Piped instance will act as the middleman between you and YouTube. These instances are located at different physical locations - indicated by their country flag(s). Instances physically closer to you are likely faster than instances that are not. You can change the instance you use in the settings at any time.</string>
    <string name="registration_disabled">Registration disabled</string>
```
```suggestion
    <string name="choose_instance_long">Please choose a Piped instance to use from below. The Piped instance will act as the middleman between you and YouTube. These instances are located at different physical locations - indicated by their country flag(s). Instances physically closer to you are likely faster than instances that are not. You can change the instance you use in the settings at any time.</string>
    <string name="registration_disabled">Registration disabled</string>
```

Reply:
Sounds better.
'- indicated by their country flag(s) -.'
(I dont know in english, but maybe you missed the ending hyphen)